perl/lib/Seq.pm

```diff
@@ -176,160 +196,162 @@ sub annotateFile {
 
   my $outJson = $self->outputJson;
 
-  mce_loop_f {
-    #my ($mce, $slurp_ref, $chunk_id) = @_;
-    #    $_[0], $_[1],     $_[2]
-    open my $MEM_FH, '<', $_[1];
-    binmode $MEM_FH, ':raw';
+  for my $inFh (@$inFhs) {
+    mce_loop_f {
```

Comment:
from here to line 353 (end of the mce_loop_f block) is unchanged besides being indented